### DISCUSSION TITLE
Proof of academic status won't go through

### DISCUSSION THREAD

**Discussion by MeiMei427** [community/community#56640] [GitHub Education]
### Hi everyone,

why can't I upload the proof of my academic status, I tried my ID card, transcripts, i20, and enrollment verification form generated by my school.  The upload page always gives me this error:  "Your application cannot be reviewed until you fix the following: please upload proof of your current school affiliation." Anyone can help me here? Thanks a lot!

**Comment by Ntshangase** [comment]
Hi @MeiMei427 

It looks like your application has been approved, I think the proof of academic status went through. you can confirm here: [Click me](https://education.github.com/discount_requests/application)

**Comment by samus-aran** [comment]
Hi @MeiMei427 👋 

Apologies for the delayed response from the team, we hope you were able to resolve your rejection reasons and receive a successful verdict on your application. I'll be closing this discussion due to lack of recent activity. If you still haven't received your Global Campus benefits please feel free to re-open and we'll be happy to assist!
